localstack/services/apigateway/context.py

```diff
@@ -131,3 +131,7 @@ def is_websocket_request(self):
     def is_v1(self):
         """Whether this is an API Gateway v1 request"""
         return self.apigw_version == ApiGatewayVersion.V1
+
+    def cookies(self):
+        cookies = self.headers.get("cookie") or ""
+        return [cookie for cookie in cookies.split(";")]
```

Comment:
Should we return a dict with the parsed cookie key/value pairs here?

We could use the `SimpleCookie` built-in utils - something like this should work (not tested, though!):
```
from http.cookies import SimpleCookie

cookie = SimpleCookie()
cookie.load(cookie_string)
cookies = {}
for key, morsel in cookie.items():
    cookies[key] = morsel.value
```

Reply:
Edit API Gateway events return unparsed cookie values as a list of strings, so this should be good for now 👍  https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html

We can extend it later on, if the actual cookie values become relevant to parse (which doesn't seem to be the case thus far..).

Reply:
As discussed in private, this is to pass cookies into the integration and an array is expected.